File: vm_Assignments/vmc/L2.5/AverageOfSquares.c
#include<stdio.h>
float average_of_squares0(float* n)
{
    float sum=0;
    for(int i=0; i<*n; i++)
        sum = sum + (i*i);
    float av = sum/(*n - 1);
    return av;
}
float average_of_squares1(float* n)
{
    float sum=0;
    for(int i=1; i<=*n; i++)
        sum = sum + (i*i);
    float av = sum/(*n);
    return av;
}

int main()
{
    float i = 3;
    printf("%f\n", average_of_squares0(&i));
    printf("%f", average_of_squares1(&i));
}

File: vm_Assignments/vm/Beginning/L2.5/AverageOfSquares.py
def average_of_squares0(n):
    sum = 0
    for i in range(n):
        sum = sum + (i*i)
    av = sum/(n-1)
    print(av)

def average_of_squares1(n):
    sum = 0
    for i in range(n+1):
        sum = sum + (i*i)
    av = sum/(n)
    print(av)

average_of_squares1(4)
average_of_squares0(5)